packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -434,7 +434,7 @@ function updateMemoComponent(
       Component.type,
       null,
       nextProps,
-      null,
+      workInProgress,
```

Comment:
Why were we passing `null` before?

Reply:
Not sure. This code was ~2 years old (#13903).

cc @sebmarkbage in case `null` was intentional/important.